sdk/lib/drivers/arbiter/arbiter.c

```diff
@@ -34,10 +230,135 @@ ArbInitializeArbiterInstance(
 {
     NTSTATUS Status;
 
-    DPRINT("ArbInitializeArbiterInstance: Initializing %S Arbiter\n", ArbiterName);
-    UNIMPLEMENTED;
+    DPRINT("ArbInitializeArbiterInstance: '%S'\n", ArbiterName);
+
+    ASSERT(Arbiter->UnpackRequirement != NULL);
+    ASSERT(Arbiter->PackResource != NULL);
+    ASSERT(Arbiter->UnpackResource != NULL);
+
+    ASSERT(Arbiter->MutexEvent == NULL &&
+           Arbiter->Allocation == NULL &&
+           Arbiter->PossibleAllocation == NULL &&
+           Arbiter->AllocationStack == NULL);
+
+    Arbiter->Signature = 'sbrA';
```

Comment:
Please, create defines for signatures and allocation tags

Reply:
Ok